cypress/integration/visitorCanViewAllListedArticles.feature.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-undef */
+describe("Visitor can see list of articles", () => {
+  describe("successfully", () => {
+    beforeEach(() => {
+      cy.server();
+      cy.route({
+        method: "GET",
+        url: "http://localhost:3000/api/articles",
+        response: "fixture:articles_data.json",
+      });
+      cy.visit("/");
+    });
+
+    it("successfully view all listed articles", () => {
+      cy.get('[data-cy="index"]').within(() => {
+        cy.contains("Title1");
+        cy.contains("Title2");
+        cy.contains("Title3");
```

Comment:
I would like to see that you are testing for all the data that you display, not just the title

Reply:
Yes! We will do so. Thanks you.